include/NovelRT/Maths/Utilities.h

```diff
@@ -5,12 +5,19 @@
 #define NOVELRT_MATHS_UTILITIES_H
 
 #include <NovelRT/Exceptions/InvalidOperationException.h>
+#include <array>
 
 namespace NovelRT::Maths
 {
     class Utilities
     {
     public:
+        [[nodiscard]] inline constexpr static std::array<uint8_t, 32> Log2DeBruijin() noexcept
```

Comment:
Just calling out, before I forget, we'll want to add proper attribution here to both .NET Runtime and http://graphics.stanford.edu/~seander/bithacks.html

Reply:
So uhh...can I get just a quick explanation on what this is?

Reply:
Seems I spelt Debruijn wrong here.